apps/studio/components/interfaces/Storage/AnalyticsBuckets/AnalyticsBucketDetails/NamespaceWithTables/TableRowComponent.tsx

```diff
@@ -336,22 +345,28 @@ export const TableRowComponent = ({
                 </DropdownMenuContent>
               </DropdownMenu>
             </>
-          </TableCell>
-        )}
+          ) : (
+            <>
+              <Badge variant="success" className="uppercase">
+                New
+              </Badge>
+            </>
+          )}
+        </TableCell>
       </TableRow>
       <ConfirmationModal
         size="medium"
         variant="warning"
         visible={showStopReplicationModal}
         loading={isUpdatingReplication}
-        title="Confirm to disable replication for table"
+        title="Disable replication on table"
         confirmLabel="Disable replication"
         onCancel={() => setShowStopReplicationModal(false)}
         onConfirm={() => onConfirmStopReplication()}
       >
         <p className="text-sm text-foreground-light">
-          Data within the "{table.name}" table will stop replicating. However do note that,
-          re-enabling replication on this table will clear and re-sync all data in it. Are you sure?
+          Data within the "{table.name}" table will stop replicating. Re-enabling replication on
+          this table will clear and re-sync all data in it. Are you sure?
```

Comment:
Minor copywriting simplification: remove redundant words to simplify overall dialog